doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/event-listeners-timing-con.adoc

```diff
@@ -0,0 +1,21 @@
+[id='event-listeners-timing-con_{context}']
+= Timing of calls to event listeners
+
+A number of event listener calls are `before` and `after` events, for example, `beforeNodeLeft()` and `afterNodeLeft()`, `beforeTaskActivatedEvent()` and `afterTaskActivatedEvent()`.
```

Comment:
```suggestion
A number of event listener calls are `before` and `after` events, for example, `beforeNodeLeft()` and `afterNodeLeft()`, `beforeTaskActivatedEvent()`, and `afterTaskActivatedEvent()`.
```

Reply:
The idea is that "`beforeNodeLeft()` and `afterNodeLeft()`" is one group while `beforeTaskActivatedEvent()` and `afterTaskActivatedEvent()` are another group - is this ok?

Reply:
Oh, I misread. Its fine. Ideally, you would put an 'and' where the comma is but I think that would make it more confusing.